test/node/test_front.rb

```diff
@@ -113,6 +113,19 @@ def test_pushes_twice
     end
   end
 
+  def test_402_on_negative_balance
+    FakeHome.new.run do |home|
```

Comment:
@jedediah the same question - does tax debt always mean negative balance?

Reply:
This test only covers negative balance, not tax debt.

Reply:
@jedediah I'm confused, the issue is about tax debt, but you focus on negative balance?